dmscripts/notify_buyers_to_award_closed_briefs.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, date, timedelta
+
+import dmapiclient
+from dmutils.email.dm_notify import DMNotifyClient
+from dmutils.email.exceptions import EmailError
+from dmutils.formats import DATE_FORMAT
+
+from dmscripts.helpers import logging_helpers, date_helpers, brief_data_helpers
+from dmscripts.helpers.logging_helpers import logging
+
+logger = logging_helpers.configure_logger({'dmapiclient': logging.INFO})
+
+
+def _filter_brief_users_by_optional_user_id_list(brief, user_id_list):
+    """Filter the brief's users by restricted user ID list if present
+    """
+    if user_id_list:
+        return filter(lambda user: user['id'] in user_id_list, brief['users'])
+    else:
+        return brief['users']
+
+
+def _filter_briefs_by_optional_user_id_list(closed_briefs, user_id_list):
+    """Return briefs if they have any users
```

Comment:
A brief should always have a user. There is a small chance that someone exists who is inactive and responsible for a brief but there should always exist someone.

With that being the case, we wouldn't always need to call this and it would be much clearer that we only need to call this function if a user_id_list is present i.e. a user_id_list is not optional.